esper/controllers/enterprise/group.py

```diff
@@ -700,3 +713,82 @@ def devices(self):
                     }
                 )
             self.app.render(devices, format=OutputFormat.JSON.value)
+
+    @ex(
+        help='Move a group',
+        arguments=[
+            (['-g', '--group'],
+             {'help': 'Group name',
+              'action': 'store',
+              'dest': 'group'}),
+            (['-p', '--parent'],
+             {'help': 'Parent Id',
+              'action': 'store',
+              'dest': 'parent'}),
+            (['-j', '--json'],
+             {'help': 'Render result in Json format',
+              'action': 'store_true',
+              'dest': 'json'})
+        ]
+    )
+    def move(self):
+        validate_creds_exists(self.app)
+        db = DBWrapper(self.app.creds)
+        group_client = APIClient(db.get_configure()).get_group_api_client()
+        enterprise_id = db.get_enterprise_id()
+        action = 'move'
+
+        if self.app.pargs.group:
+            group_name = self.app.pargs.group
+            kwargs = {'name': group_name}
+            try:
+                search_response = group_client.get_all_groups(enterprise_id, limit=1, offset=0, **kwargs)
```

Comment:
group_name search may return multiple results. Because nested group feature is following unique name per level.

Reply:
I'll accept group id from user instead?